misc/build.func

```diff
@@ -108,19 +148,31 @@ fi
 # This function checks the system architecture and exits if it's not "amd64".
 arch_check() {
   if [ "$(dpkg --print-architecture)" != "amd64" ]; then
-    echo -e "\n ${CROSS} This script will not work with PiMox! \n"
-    echo -e "\n Visit https://github.com/asylumexp/Proxmox for ARM64 support. \n"
+    echo -e "\n ${INFO}${YWB}This script will not work with PiMox! \n"
+    echo -e "\n ${YWB}Visit https://github.com/asylumexp/Proxmox for ARM64 support. \n"
     echo -e "Exiting..."
     sleep 2
     exit
   fi
 }
 
+# This function sets the APP-Name into an ASCII Header in Slant, figlet needed on proxmox main node.
+header_info() {
+    if ! command -v figlet &> /dev/null; then
+        apt-get install -y figlet &> /dev/null
+    fi
+    ascii_art=$(figlet -f slant "$APP")
+    clear
+    cat <<EOF
+$ascii_art
+EOF
+}
+
 # This function checks if the script is running through SSH and prompts the user to confirm if they want to proceed or exit.
 ssh_check() {
   if command -v pveversion >/dev/null 2>&1 && [ -n "${SSH_CLIENT:+x}" ]; then
```

Comment:
```suggestion
  if [ -n "${SSH_CLIENT:+x}" ]; then
```

If we are overhauling, lets remove the redundant check. It does not need to check it pveversion exists.